README.rst

```diff
@@ -0,0 +1,76 @@
+|Travis|
+|Conda|
+|PythonConda|
+|Codecov|
+|CondaVersion|
+
+QCGrids is a numerical integration library for quantum chemistry.
+
+Dependencies
+============
+
+-  celllists, python-celllists
+-  C++11 compiler (tested: GNU and Intel)
```

Comment:
Probably better to say GNU and clang. we're not systematically testing with Intel compilers but it should obviously not be hard to compile with Intel.